kubernetes/README.md

```diff
@@ -1,820 +0,0 @@
-	DNS_PREFIX=ascend-k8s
-	CLUSTER_NAME=btk8s
-	az acs create --orchestrator-type=kubernetes \
-	         --resource-group $RESOURCE_GROUP \
-	         --name=$CLUSTER_NAME --dns-prefix=$DNS_PREFIX
-	az acs kubernetes get-credentials --resource-group=$RESOURCE_GROUP --name=$CLUSTER_NAME
-
-Note that you will need to substitute your own information for what you see above. For example, the subscription ID you see will need to be your own.
-
-Anytime you see the commands include, "az acs" you should realize that this involves the Azure and Python SDK to be installed on your developer computer.
-
-You will also need to take special note with some of the environment variables you see above, substituting your own names as indicated below.
-
-#### Notes about the code snippet above
-
-You can get more information about provisioning your own Kubernetes cluster here in addition to the videos previously presented:
-
-https://docs.microsoft.com/en-us/azure/container-service/container-service-kubernetes-walkthrough
-
-### Setting environment variable (KUBECONFIG) and getting kubectl
-
-This environment variable is used by kubectl to know where to retrieve the config file.
-
-#### Lab Instructions
-
-1. Run the **export** command as seen below for KUBECONFIG
-	1. You can add this to **~/.bashrc** for a more permanent solution
-2. Run the **az acs kubernetes install-cli** as seen below, in addition to the test specified
-3. Review the following URLs for more information:
-	1. https://www.youtube.com/watch?v=qW6mzCJg4tI
-	2. https://www.youtube.com/watch?v=2316KyERwvg
-
-Here is the command that allows the system to find your **config** file
-
-	export KUBECONFIG=`pwd`/config
-
-We will also want to get the **kubectl** utility, which is the mechanism we can use to communicate with their Kubernetes cluster. It works in conjunction with the **config** file just addressed.
-
-	# obtain the kubectl utility
-	az acs kubernetes install-cli
-	# test the utility
-	kubectl cluster-info
-
-You can see the config file below being copied to the current working directory. You probably want to think about a more permanent location for the config file longer-term, but for now let's just copy it to a local directory for ease of use.
-
-It's important to note that it's this config file that will enable us to communicate with the Kubernetes cluster from our Linux VM.
-
-![](./images/kubernetes0010.png)
-
-_Figure 8: Verifying config_
- 
-----------
-
-## Part 7 - Using kubectl to run containers, provision pods, and provision services
-
-We have now laid the groundwork for using kubectl. This means that we can begin to communicate with the cluster and indicate to the cluster which containers we wish to run in which pods and how to expose the pods through services.
-
-So that I do not have to type **kubectl** over and over, I have run this following command in Linux to create an alias:
-
-	alias k=kubectl
-
-This means I can just use **k** instead of **kubectl**. 
```

Comment:
If you're going to touch on this, you might want to touch on the fact that they can do:

`source <(kubectl completion bash)` to get tab completion:

`kubectl <tab>` (shows "get", "logs", "describe", etc)

`kubectl get pods <tab>` (shows pods in the default namespace)

`kubectl get pods --namespace kube-system` (shows pods in kube-system namespace).

To be frank, those will save a LOT more time than "k" vs "kubectl".

I only bring this up because not enough people know about it.